assets/templates/tabs-component.html

```diff
@@ -0,0 +1,749 @@
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="canonical" href="" />
+    <!-- Open Graph metatags -->
+    <meta property="og:title" content="Breadcrumb component" />
+    <meta property="og:type" content="article" />
+    <meta property="og:image" content="Breadcrumb component" />
+    <meta property="og:url" content="Breadcrumb component" />
+    <meta property="og:description" content="Breadcrumb component" />
+    <meta property="og:site_name" content="Breadcrumb component" />
+    <!-- Stylesheets and polyfills -->
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
+    <!-- Favicons -->
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="144x144"
+      href="https://www.fromdoppler.com/apple-touch-icon-144x144-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="114x114"
+      href="https://www.fromdoppler.com/apple-touch-icon-114x114-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="72x72"
+      href="https://www.fromdoppler.com/apple-touch-icon-72x72-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      sizes="57x57"
+      href="https://www.fromdoppler.com/apple-touch-icon-57x57-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon-precomposed"
+      href="https://www.fromdoppler.com/apple-touch-icon-precomposed.png"
+    />
+    <link
+      rel="apple-touch-icon"
+      href="https://www.fromdoppler.com/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+    <link
+      rel="shortcut icon"
+      type="image/x-icon"
+      href="https://www.fromdoppler.com/favicon.ico"
+    />
+  </head>
+  <body data-active-menu="25">
+    <!-- Layers -->
+    <main role="main" class="dpsg-main-wrap">
+      <!-- build:include partials/_header.html -->
+      This will be replaced by the content of header.html
+      <!-- /build -->
+      <!-- header -->
+      <div class="dpsg-wrapper">
+        <!-- build:include partials/_navigation.html -->
+        This will be replaced by the content of navigation.html
+        <!-- /build -->
+        <div class="dpsg-content-wrap">
+          <article class="dpsg-content">
+            <h3>Tabs</h3>
+            <p>
+              Los tabs nos van a permitir crear una interfaz de usuario basada
+              en pestañas, donde, de una forma muy intuitiva, podemos ofrecer al
+              usuario diferentes vistas, que son seleccionadas al pulsar una de
+              las pestañas que se muestran a continuación.
+            </p>
+            <hr class="dpsg-break" />
+            <div class="dp-container">
+              <div class="dp-rowflex">
+                <nav class="wrapper-tabs">
```

Comment:
Perhaps here you could think of a more cohesive way for the naming of the classes, for example:
``` 
<nav class="tab-menu--wrapper">
      <ul class="tab-menu">
         <li class="tab-menu--item">
...
```
It gives you an idea of this tab as a component, tab-menu, which has many parts: a wrapper and items. 
I'm not so sure about `tab-menu`, perhaps another name would be ok, what do you think?

Reply:
ok, makes sense i check it